core/src/credential.rs

```diff
@@ -80,4 +94,45 @@ impl Credential {
     pub fn serialize(&self) -> Value {
         return serde_json::to_value(&self).unwrap();
     }
+
+    pub fn create_verifiable_credentials(
+        &self,
+        integrity_proof: crate::proof::DataIntegrityProof,
+    ) -> VerifiableCredential {
+        let vc = VerifiableCredential {
+            credential: self.to_owned(),
```

Comment:
if we're taking ownership of self here then we should just use `self` instead of `&self`
```suggestion
    pub fn create_verifiable_credentials(
        self,
        integrity_proof: crate::proof::DataIntegrityProof,
    ) -> VerifiableCredential {
        let vc = VerifiableCredential {
            credential: self,
```